src/components/cylc/gscan/GScan.vue

```diff
@@ -166,8 +166,6 @@ export default {
     filters: {
       'workflow state': [],
       'task state': []
-      // 'workflow host': [], // TODO: will it be in state totals?
-      // 'cylc version': [] // TODO: will it be in state totals?
```

Comment:
Dropped plans to add these filters?

Reply:
No, just removed dead end TODOs.

Reply:
To answer the question in the TODO, the info will not be in state_totals.

Reply:
I think it is useful to leave the placeholders there if there are plans to add them. Is there an issue for them?

Reply:
Issue: https://github.com/cylc/cylc-ui/issues/581

Either:

```suggestion
```

Or:

```suggestion
      // 'workflow host': [], // https://github.com/cylc/cylc-ui/issues/581
      // 'cylc version': [] // https://github.com/cylc/cylc-ui/issues/581
```

Reply:
```
      // 'workflow host': [], // https://github.com/cylc/cylc-ui/issues/581
      // 'cylc version': [] // https://github.com/cylc/cylc-ui/issues/581
```
Looks good to me